api/v1alpha1/catalogentry_types.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	kcpv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/apis/conditions/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// InternalAPIBindingEntryLabelKey is the label key on an APIBinding with the
+	// base62(sha224(<workspace>:<exportName>)) as value to filter bindings by catalog entry.
+	InternalAPIBindingEntryLabelKey = "internal.catalog.kcp.dev/entry"
```

Comment:
is this actually needed now?

Reply:
This is a label key that I intend to use for linking APIBinding to CatalogEntry for reconciliation on the controller side. However, it is not needed now so I can drop this.